mod.ts

```diff
@@ -56,6 +56,15 @@ export namespace Drash {
     export const Exports = util_members;
   }
 
+  /**
+   * @description
+   *     Drash version. Also represents what Deno version is
+   *     supported.
+   *
+   * @property string version
+   */
+  export const version: string = "v0.41.0";
```

Comment:
we're currently at v0.41.1

Reply:
ok i get what you're saying now.. so this is a problem. the bump version script will need to handle this file differently because although we match deno's version, we can still push out fixes and increment the patch version.

Reply:
Changed it to a manual change, which isn't a problem because it adds about 10s onto the update process